.circleci/config.yml

```diff
@@ -1,20 +1,20 @@
-version: 2.1
-orbs:
-  node: circleci/node@1.1.6
-jobs:
-  build-and-test:
-    executor:
-      name: node/default
-    steps:
-      - checkout
-      - node/with-cache:
-          steps:
-            - run: yarn install
-            - run: yarn run fix
-            - run: "[[ -z $(git --no-pager diff -- . ':(exclude)yarn.lock' ':(exclude)yarn.lock') ]]"
-            - run: yarn run test --passWithNoTests
-            - run: export CI=1
-workflows:
-    build-and-test:
-      jobs:
-        - build-and-test
+version: 2.1
+orbs:
+  node: circleci/node@1.1.6

```

Comment:
there are changes in all these random files, please make the PR clean and then PTAL again

Reply:
actually, there is no change in these files, I don't know why they appear in the pr

Reply:
maybe google "windows git newline endings" but either way they should be removed. it makes it hard for the reviewer to know what should and shouldn't be removed.

google: "how to remove a file from a pull request" or "how to checkout git file from another branch" (you will want to check out from dev)

Reply:
should I close this pr and create a new one without those unchanged files?